salt/modules/cp.py

```diff
@@ -764,15 +777,16 @@ def push_dir(path, glob=None, upload_path=None):
         return push(path, upload_path=upload_path)
     else:
         filelist = []
-        for root, dirs, files in os.walk(path):
+        for root, files in os.walk(path):
```

Comment:
That was removed due to pylint warnings about dirs being an unused variable. The failure is on me for trusting the linting (Not really used to linting) and not looking up the documentation for os.walk.

I'll commit a fix.